examples/example-tls/src/main/java/io/grpc/examples/helloworldtls/HelloWorldClientS2A.java

```diff
@@ -95,20 +93,32 @@ private static HelloWorldClientS2A create(String host, int port, SslContext sslC
      * Sends a {@code HelloRequest} to a greeter service.
      *
      * args:
-     * [ServerAddress] In the form of host:port
-     * [clientCertChainFilePath] File Path to Client Certificate
-     * [clientPrivateKeyFilePath] File Path to Client Key
-     * [trustCertCollectionFilePath] File Path to CA Certificate
+     * @param arg[0] contains the server address in the form of host:port
+     * @param arg[1] contains the file path to the client certificate
+     * @param arg[2] contains the file path to the client's private key
+     * @param arg[3] contains the file path to the CA certificate
      */
     public static void main(String[] args) throws SSLException, InterruptedException {
+        String[] defaultArgs = new String[] {
+          "localhost:50051",
+          "testdata/client.pem",
+          "testdata/client.key",
+          "testdata/ca.pem"};
+
+        //Check if the args are missing information
```

Comment:
Please leave a space between `//` and `Check`. Please also change `Check` to `Checks`, and add a period at the end of the sentence.